src/api/config.ts

```diff
@@ -13,23 +11,13 @@ const resolveApiBaseUrl = (defaultBaseUrl = ''): string => {
     return 'http://localhost';
   }
 
-  // use a known URL for development
-  if (
-    typeof window !== 'undefined' &&
-    process.env.NODE_ENV === 'development' &&
-    typeof process.env.NEXT_PUBLIC_API_HOST_CLIENT === 'string'
-  ) {
-    return process.env.NEXT_PUBLIC_API_HOST_CLIENT;
+  // Client-side: use NEXT_PUBLIC_API_HOST
```

Comment:
I'm not able to test out in localhost, the API host becomes localhost. Was this intentional?